src/api/__tests__/campaigns-test.js

```diff
@@ -147,11 +147,13 @@ describe('campaigns', function() {
       var callback = jest.genMockFunction();
       campaigns.leaderboard('xy-123', 'abcd', 'foo', 12, callback, {
         includePages: true,
-        includeFootprint: true
+        includeFootprint: true,
+        groupValue: 'ABC&group_value[]=DEF'
```

Comment:
This looks a little like confirming the existence of a hack?
Shouldn't this be: `groupValues: ['ABC', 'DEF']`?